tests/imfile-file-not-found-error.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# add 2016-11-22 by Pascal Withopf, released under ASL 2.0
+. $srcdir/diag.sh init
+. $srcdir/diag.sh generate-conf
+. $srcdir/diag.sh add-conf '
+module(load="../plugins/imfile/.libs/imfile")
+
+input(type="imfile" File="testsuites/NotExistingInputFile" Tag="tag1")
+
+action(type="omfile" file="rsyslog.out.log")
+'
+. $srcdir/diag.sh startup
+. $srcdir/diag.sh shutdown-when-empty
+. $srcdir/diag.sh wait-shutdown
+
+grep "imfile: error.*No such file or directory" rsyslog.out.log > /dev/null
```

Comment:
The file name should also be checked in the test. I think it is mandatory that it is given, else the user gets confused.